app/src/main/java/com/example/sharingang/NewItemFragment.kt

```diff
@@ -124,5 +163,8 @@ class NewItemFragment : Fragment() {
     private fun updateLocation(location: Location) {
         binding.latitude = location.latitude.toString()
         binding.longitude = location.longitude.toString()
+        val address =
+            geocoder.getFromLocation(location.latitude, location.longitude, 1).getOrNull(0)
+        binding.postalAddress.text = if(address!=null) address.getAddressLine(0) else ""
```

Comment:
```suggestion
        binding.postalAddress.text = address?.getAddressLine(0) ?: ""
```

I think this works, and is a bit more Kotlinesque

Reply:
Way cleaner, thanks!